plugins/labhub.py

```diff
@@ -96,7 +96,7 @@ def TEAMS(self, new):
         self._teams = new
 
     # Ignore LineLengthBear, PycodestyleBear
-    @re_botcmd(pattern=r'(?:(?:invite)|(?:inv))\s+(?:(?:@?([\w-]+)(?:\s*(?:to)\s+(\w+))?)|(me))',
+    @re_botcmd(pattern=r'^(?:(?:invite)|(?:inv))\s+(?:(?:@?([\w-]+)(?:\s*(?:to)\s+(\w+))?)|(me))',
```

Comment:
You should also mark `$` in the end of regex to show the end of command.

Reply:
I thought of it but saw line `172`

they have not used it there.

So it is not necessary I guess and it was working fine also
for maintaining continuity in the project, I didn't add it.

Reply:
https://github.com/yashovardhanagrawal/corobo/blob/0945e06d95cf6ea9570a2d848c54691a699fa4e6/plugins/labhub.py#L172

Reply:
also in [line 247](https://github.com/yashovardhanagrawal/corobo/blob/0945e06d95cf6ea9570a2d848c54691a699fa4e6/plugins/labhub.py#L247) you can view it

Reply:
Yeah! we need to change those regex also to a full match. We don't want any other text after the command.

Reply:
should I change in this PR also.. I am a bit skeptical about that.. let a maintainer comment on this 😛

Reply:
Yeah, add `$` at the end